### PR TITLE
Coral credits installation

### PR DESCRIPTION
**Pull request by assumptionsandg**
https://github.com/azimuth-cloud/azimuth-config/pull/158 config patch

### CODE REVIEW

File: roles/coral_credits/defaults/main.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+coral_credits_release_namespace: coral-credits
+coral_credits_release_name: coral-credits
+
+# The timeout to wait for coral credits API to become ready
+coral_credits_wait_timeout: 10m
+
+# The values for the release
+coral_credits_release_defaults: {}
+coral_credits_release_overrides: {}
```
Comment (JohnGarbutt) [inline roles/coral_credits/defaults/main.yaml:15]: We need to make sure this requires the setting of the admin password, similar to what we do in other roles please.

**Review by JohnGarbutt — CHANGES_REQUESTED** [review]

File: roles/coral_credits/defaults/main.yaml (hunk 1 above)
Comment (JohnGarbutt) [reply roles/coral_credits/defaults/main.yaml:15]: copy this style please:
https://github.com/stackhpc/ansible-collection-azimuth-ops/blob/500e98bc4ff8c93e8a75207f9908e483a737c866/roles/admin_dashboard_ingress/defaults/main.yml#L40

Comment (JohnGarbutt) [reply roles/coral_credits/defaults/main.yaml:15]: We should do that in a style similar to Zenith, where we default to ingress being correctly configured, with TLS as appropriate:
https://github.com/stackhpc/ansible-collection-azimuth-ops/blob/500e98bc4ff8c93e8a75207f9908e483a737c866/roles/zenith/defaults/main.yml#L85-L143

File: roles/coral_credits/defaults/main.yaml [hunk 2]
```diff
@@ -0,0 +1,101 @@
+---
+
+# The chart to use
+coral_credits_chart_repo: https://stackhpc.github.io/coral-credits
+coral_credits_chart_name: coral-credits
+coral_credits_chart_version: 0.3.5
+
+# Release information for the coral credits release
+coral_credits_release_namespace: "{{ azimuth_release_namespace | default('azimuth') }}"
```
Comment (mkjpryor) [inline roles/coral_credits/defaults/main.yaml:9]: Can we put coral-credits in its own namespace by default please?

File: roles/coral_credits/defaults/main.yaml [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+coral_credits_release_name: coral-credits
+
+# The timeout to wait for coral credits API to become ready
+coral_credits_wait_timeout: 10m
+
+# Feature flags
+#   Indicates if coral credits should be enabled
+coral_credits_enabled: yes
```
Comment (mkjpryor) [inline roles/coral_credits/defaults/main.yaml:17]: Should default to `no` I think?

**Review by mkjpryor — CHANGES_REQUESTED** [review]
Just a couple of changes

File: roles/coral_credits/defaults/main.yaml [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+# Ingress settings
+#   The hostname for Coral credits ingress resources
+coral_credits_ingress_host: >-
+  {{-
+    "{}.{}".format(coral_credits_ingress_subdomain, ingress_base_domain)
+    if coral_credits_ingress_subdomain is defined and ingress_base_domain is defined
+    else undef(hint = 'coral_credits_ingress_subdomain is required')
+  }}
+#   Subdomain for the Coral credits ingress host
+coral_credits_ingress_subdomain: "coral"
```
Comment (JohnGarbutt) [inline roles/coral_credits/defaults/main.yaml:53]: I remember now, we should probably make this "credits" instead of "coral"?

**Review by JohnGarbutt — CHANGES_REQUESTED** [review]

File: roles/coral_credits/defaults/main.yaml (hunk 3 above)
Comment (assumptionsandg) [reply roles/coral_credits/defaults/main.yaml:17]: We can set it to no here, was leaving it enabled to for it to run in CI.

It's also disabled in the config here https://github.com/azimuth-cloud/azimuth-config/pull/158/files

File: roles/coral_credits/defaults/main.yaml (hunk 2 above)
Comment (assumptionsandg) [reply roles/coral_credits/defaults/main.yaml]: Set default to coral-credits

File: roles/coral_credits/defaults/main.yaml (hunk 3 above)
Comment (assumptionsandg) [reply roles/coral_credits/defaults/main.yaml]: Disabled

File: roles/coral_credits/defaults/main.yaml (hunk 4 above)
Comment (assumptionsandg) [reply roles/coral_credits/defaults/main.yaml]: Changed to credits subdomain

**Pull request merged**